packages/nimble-components/src/combobox/specs/IxD.md

```diff
@@ -0,0 +1,241 @@
+
+> THE VISUAL DESIGN IN THIS DOCUMENT MAY NOT BE ACCURATE
+
+### Background
+
+-   [Select and Combobox Refresh HLD (#1303)](https://github.com/ni/nimble/issues/1303)
+-   [ViD Combobox Styles (Figma)](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1529-41777&mode=design)
+
+## Usage
+
+When to use:
+
+-   To provide an input for text entry coupled with a popup for quick selection and filtering
+-   To allow end-users to enter custom values
+-   For entering settings or preference values that don't (yet) exist
+-   For single selection
+
+When not to use:
+
+-   When a value that is not in the list of options needs to be entered
+
+-   When there are fewer than three options
+-   For complex and/or hierarchical data
+-   For multi selection
+
+### Anatomy
+
+![ ](spec-images/combobox-1.png)
+
+| Element         | Description                                                                 |
+| --------------- | --------------------------------------------------------------------------- |
+| Label           | Text that tells the end-user what to expect in the component                |
+| Field input     | Input that displays the current selected or entered option                  |
+| Dropdown button | Click area that opens the dropdown                                          |
+| Dropdown        | A list of options to choose from displayed as an _open_ state               |
+| List item       | A selection the end-user can make, shown with other options in the dropdown |
+
+#### Label
+
+Comboboxes should always have a label, unless approved by an interaction designer or user researcher (usually in toolbar use cases). _See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information._
+
+#### Field Input
+
+Comboboxes can be empty by default or have a default selection from the menu. Empty comboboxes should include placeholder text that propts the end-user to enter a custom value or select an option from the menu. Placeholder text should always follow the pattern "Enter or select [thing(s)]", for example "Enter or select address". Ellipses are not needed for the placeholder. Use sentence casing for the placeholder.
+
+![ ](spec-images/combobox-2.png)
+
+#### Dropdown & List Items
+
+A list of options to select, displayed as an open state._See [Select](/packages/nimble-components/src/select/specs/IxD.md) for more information._
+
+### Related Components
+
+To allow end-users to select a list item but not enter a custom value, use a [**select**](/packages/nimble-components/src/select/specs/IxD.md).
+
+![ ](spec-images/combobox-alternative-select.png)
+
+To allow end-users to multi-select list items, use a [**tag picker**](/packages/nimble-components/src/tag-picker/specs/IxD.md).
+
+![ ](spec-images/combobox-alternative-picker.png)
+
+To allow end-users to select less than three options, use a [**radio group**](/packages/nimble-components/src/radio-group/specs/IxD.md).
+
+## Design
```

Comment:
I don't see any discussion of error states. I'm assuming we keep the ability to put the control in an error state and show an error message below the control?

Reply:
Added, and you are correct.